lib/index.js

```diff
@@ -170,7 +169,7 @@ export default function runContentfulExport (params) {
         console.log('No data was exported')
       }
 
-      if ('assetDownloads' in summary) {
+      if ('assetDownloads' in ctx) {
```

Comment:
A small bugfix I sneaked in. The assets table actually was never shown. Now it does :)